src/components/InputField.vue

```diff
@@ -117,9 +139,153 @@ async function keyDown(key: KeyboardEvent) {
         emit("direction", "down");
       }
     }
-  } else if (key.key == "Backspace") {
-    crosswordTile.value.currentLetter = "";
+  } else {
+    switch (key.key) {
+      case "ArrowRight": {
+        pressRightArrow();
+        emit("direction", "right");
+        break;
+      }
+      case "ArrowLeft": {
+        emit("direction", "right");
+        pressLeftArrow();
+        break;
+      }
+      case "ArrowDown": {
+        emit("direction", "down");
+        pressDownArrow();
+        break;
+      }
+      case "ArrowUp": {
+        emit("direction", "down");
+        pressUpArrow();
+        break;
+      }
+      case "Tab": {
+        pressTab();
+        break;
+      }
+    }
```

Comment:
What about extracting this switch into a separate method `handleButtonPress` or similar?